package testfiles.DefnVals.WithoutDeclaredType.InClass.Inferrable.TermApply.WithTermName.Failure;

import io.vavr.control.Try;
import io.vavr.control.Try.failure;

public class Sample {
    private final Try<RuntimeException> x = failure(new RuntimeException());
}